src/labels/CustomLabels.labels

```diff
@@ -2127,6 +2135,38 @@
         <shortDescription>Plural Picklist substitute for Yearly</shortDescription>
         <value>Years</value>
     </labels>
+    <labels>
+        <fullName>RD2_EntryFormSaveCommitmentMessage</fullName>
```

Comment:
Does it make sense to remove "Commitment" from the label name too?

Reply:
These custom labels should show Recurring Donation save status:
- Credit card validation
- Recurring Donation being saved and
- Commitment being created.
The referenced custom label is for the Commitment creation save status. Not sure how else would I name it?

Reply:
What I mean is, should the fullName be`RD2_EntryFormSaveReccuringDonationMessage` instead of `RD2_EntryFormSaveCommitmentMessage`?